test/userController.js

```diff
@@ -8,7 +8,8 @@ const user = {
   firstName: 'FirstName',
   lastName: 'LastName',
   email: 'test@wolox.com.ar',
-  password: 'passwordTest1'
+  password: 'passwordTest1',
+  admin: false
```

Comment:
If you set a defaultValue for the admin column like i suggested in a previous comment, remember that this field won't be necessary